cmd/runmqdevserver/keystore.go

```diff
@@ -65,11 +65,23 @@ func (ks *KeyStore) Create() error {
 			stashFile := ks.Filename[0:len(ks.Filename)-len(extension)] + ".sth"
 			rdbFile := ks.Filename[0:len(ks.Filename)-len(extension)] + ".rdb"
 			crlFile := ks.Filename[0:len(ks.Filename)-len(extension)] + ".crl"
-			os.Remove(stashFile)
-			os.Remove(rdbFile)
-			os.Remove(crlFile)
+			err = os.Remove(stashFile)
+			if err != nil {
+				log.Debugf("Error removing %s: %v", stashFile, err)
+			}
+			err = os.Remove(rdbFile)
+			if err != nil {
+				log.Debugf("Error removing %s: %v", rdbFile, err)
+			}
+			err = os.Remove(crlFile)
+			if err != nil {
+				log.Debugf("Error removing %s: %v", crlFile, err)
```

Comment:
I think these should be regular log messages, not debug ones.  Also, in the case of removing files failing, maybe we should just fail.